core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/ssp_config.h

```diff
@@ -108,7 +108,17 @@
 #endif
 
 #if !defined(BH_PLATFORM_LINUX_SGX)
-#if defined(__GNUC_PREREQ)
+/* Clang's __GNUC_PREREQ macro has a different meaning than GCC one,
+so we have to handle this case specially */
+#if defined(__clang__)
+/* Clang provides stdatomic.h since 3.6.0
+See https://releases.llvm.org/3.6.0/tools/clang/docs/ReleaseNotes.html */
+#if __clang_major__ >= 3 || (__clang_minor__ == 3 && __clang_minor__ >= 6)
```

Comment:
This line is invalid? Should change
```C
#if __clang_major__ >= 3 || (__clang_minor__ == 3 && __clang_minor__ >= 6)
```
to:
```C
#if __clang_major__ > 3 || (__clang_major__ == 3 && __clang_minor__ >= 6)
```

Reply:
Uh, The original code works, but your version is better. Thanks for your careful inspection.